packages/teleport/src/Console/useOnExitConfirmation.test.ts

```diff
@@ -13,17 +13,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { BroadcastChannel } from 'broadcast-channel';
 
 import renderHook from 'design/utils/renderHook';
 
-import session from 'teleport/services/websession';
+import { WebSession } from 'teleport/services/websession';
 
 import ConsoleContext from './consoleContext';
 import useOnExitConfirmation from './useOnExitConfirmation';
 
 test('confirmation dialog before terminating an active ssh session', () => {
   const ctx = new ConsoleContext();
-  const { current } = renderHook(() => useOnExitConfirmation(ctx));
+
+  jest.mock('broadcast-channel');
```

Comment:
Can you move this mock into a shared test util as the same mock needs to happen across multiple files. That will also be a good spot to put the comment as to why this is necessary.